src/app/containers/OnDemandHeading/index.jsx

```diff
@@ -17,16 +26,29 @@ const BrandTitle = styled.span`
   word-break: break-word;
   ${({ script, darkMode }) => (darkMode ? '' : script && getParagon(script))}
   ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
-    padding-bottom: 0;
+    padding-bottom: ${GEL_SPACING_DBL};
     word-break: break-word;
-    line-height: ${GEL_SPACING_SEPT};
+    line-height: ${GEL_SPACING_QUAD};
   }
 `;
 
 const Subheading = styled.span`
+  display: inline-block;
+  margin: 0;
   ${({ script }) => script && getDoublePica(script)}
   ${({ service }) => getSansRegular(service)}
-  margin: 0;
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
+    font-size: ${GEL_SPACING_TRPL};
+    line-height: 1.75rem;
+  }
+  ${GEL_GROUP_2_SCREEN_WIDTH_MAX} {
```

Comment:
I think this will become `37.4375rem {` instead of `@media (max-width: 37.4375rem) {`, which means I'm not sure the CSS inside this block will behave expectedly 👍